org.metaborg.spoofax.shell.core/ruleset.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<ruleset name="Custom ruleset"
+    xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 http://pmd.sourceforge.net/ruleset_2_0_0.xsd">
+  <description>
+  Overriding some rule properties.
+  </description>
+
+  <rule ref="rulesets/java/empty.xml/EmptyCatchBlock">
+    <properties>
+      <property name="allowCommentedBlocks" value="true"/>
```

Comment:
For this line: https://github.com/spoofax-shell/spoofax-shell/pull/20/files#diff-94259d4a4971574c51859b462d3317a7R139